engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/repository-details-info.controller.js

```diff
@@ -36,6 +37,14 @@ angular.module('Bastion.repositories').controller('RepositoryDetailsInfoControll
 
         $scope.progress = {uploading: false};
 
+        Setting.get({"search": "name = enable_deferred_download_policies"},
+        function (data) {
+            data = angular.fromJson(data);
```

Comment:
Did you find this was required to parse like this? The data didn't come parse already?